paasta_tools/vitesscluster_tools.py

```diff
@@ -48,8 +564,102 @@ def __init__(
+        paasta_pool = self.get_pool()
+        paasta_cluster = self.config_dict.get("paasta_cluster", self.get_cluster())
+        cells = self.config_dict.get("cells")
+        region = self.config_dict.get("region")
+        vtgate_resources = self.config_dict.get("vtgate_resources")
+        return [
+            get_cell_config(cell, paasta_pool, paasta_cluster, region, vtgate_resources)
+            for cell in cells
+        ]
+
+    def get_vitessDashboard(self) -> Dict[str, str]:
+        paasta_pool = self.get_pool()
+        paasta_cluster = self.config_dict.get("paasta_cluster", self.get_cluster())
+        cells = self.config_dict.get("cells")
+        zk_address = self.config_dict.get("zk_address")
+        vtctld_resources = self.config_dict.get("vtctld_resources")
+        return get_vitess_dashboard_config(
+            cells, paasta_pool, paasta_cluster, zk_address, vtctld_resources
+        )
+
+    def get_vtadmin(self) -> Dict[str, str]:
+        paasta_pool = self.get_pool()
+        paasta_cluster = self.config_dict.get("paasta_cluster", self.get_cluster())
+        cells = self.config_dict.get("cells")
+        vtadmin_resources = self.config_dict.get("vtadmin_resources")
+        return get_vt_admin_config(
+            cells, paasta_pool, paasta_cluster, vtadmin_resources
+        )
+
+    def get_keyspaces(self) -> List[Dict[str, str]]:
+        paasta_pool = self.get_pool()
+        paasta_cluster = self.config_dict.get("paasta_cluster", self.get_cluster())
+        cells = self.config_dict.get("cells")
+        zk_address = self.config_dict.get("zk_address")
+        region = self.config_dict.get("region")
+        keyspaces = self.config_dict.get("keyspaces")
+        return get_keyspaces_config(
+            cells, keyspaces, paasta_pool, paasta_cluster, zk_address, region
+        )
+
+    def get_updateStrategy(self) -> Dict[str, str]:
+        return self.config_dict.get("updateStrategy", {"type": "Immediate"})
+
+    def get_vitess_config(self) -> Dict[str, str]:
+        smartstack_config = load_service_namespace_config(
+            service=self.service, namespace=self.get_namespace(), soa_dir=self.soa_dir
+        )
+        return {
+            "autoscaling": self.get_autoscaling_params(),
+            "namespace": self.get_namespace(),
+            "cpus": self.get_cpus(),
+            "mem": self.get_mem(),
+            "deploy_group": self.get_deploy_group(),
+            "healthcheck_cmd": self.get_healthcheck_cmd(),
+            "healthcheck_mode": self.get_healthcheck_mode(smartstack_config),
+            "healthcheck_grace_period_seconds": self.get_healthcheck_grace_period_seconds(),
+            "env": {
+                "OPERATOR_NAME": "vitess-operator",
+                "POD_NAME": "vitess-k8s",
+                "PS_OPERATOR_POD_NAME": "vitess-k8s",
+                "PS_OPERATOR_POD_NAMESPACE": KUBERNETES_NAMESPACE,
+                "WATCH_NAMESPACE": KUBERNETES_NAMESPACE,
```

Comment:
i think we might want to hardcode that `get_namespace` return `KUBERNETES_NAMESPACE` - otherwise, these values would be incorrect

Reply:
https://github.com/Yelp/paasta/pull/3889/commits/6cbc15067cdfc3754ceaf751a8cbe8d8ee749b7b